packages/react-native-codegen/src/parsers/parser.js

```diff
@@ -24,13 +24,17 @@ export interface Parser {
   typeParameterInstantiation: string;
 
   /**
-   * Given a property or an index declaration, it returns the key name.
-   * @parameter propertyOrIndex: an object containing a property or an index declaration.
+   * Given a declaration, it returns true if it is a property
+   */
+  isProperty(property: $FlowFixMe): boolean;
+  /**
+   * Given a property declaration, it returns the key name.
+   * @parameter propertyOrIndex: an object containing a property declaration.
```

Comment:
from @christophpurrer:
```suggestion
   * @parameter property: an object containing a property declaration.
```

Reply:
Fixed